enhancements/monitoring/metrics-collection-profiles.md

```diff
@@ -185,11 +192,12 @@ all.
 Once set up, the implementation within the operator that defines its behavior
 for such profiles will decide how it reconciles under such conditions.
 
-The goal is to support 2 profiles:
+The goal is to support 3 profiles:
 
 - `Full` (same as today)
 - `Minimal` (only collect metrics necessary for recording rules, alerts,
   dashboards, HPA, VPA and telemetry)
+- `Telemetry` (only collect metrics that are required for telemetry)
```

Comment:
do we already have a link in the doc to https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/support/remote-health-monitoring-with-connected-clusters?